Adyen/UI/Form/FormView.swift

```diff
@@ -7,16 +7,27 @@
 import UIKit
 
 /// Displays a form for the user to enter details.
-internal final class FormView: UIScrollView {
+internal final class FormView: UIView {
```

Comment:
Can we make `FormView ` a `UIStackView` instead, so we don't need to bother about constrains ?

Reply:
I tried, but sadly we rely on the `minimalSize` of the stack view. Having `FormView` as a `UIStackView` causes an infinite loop when calculating its `intrinsicContentSize`. It's a great suggestion that I will definitely do once we get rid of `minimalSize` calculations across the codebase. Thanks Vlad!